src/features/game/components/Lore.tsx

```diff
@@ -47,6 +48,11 @@ export const Lore: React.FC = () => {
       {showGreenBook && (
         <Modal centered show={showGreenBook} onHide={() => onCloseBook()}>
           <Panel>
+            <img
+              src={close}
+              className="h-6 top-4 right-4 absolute cursor-pointer"
+              onClick={() => onCloseBook()}
```

Comment:
Purley a nitpick, so take it or leave it but the *best* way to handle function calls on event handlers in react is `onClick={onCloseBook}`

for reference: https://upmostly.com/tutorials/react-onclick-event-handling-with-examples

Reply:
> Purley a nitpick, so take it or leave it but the "best" way to handle function calls without parameters on event handlers in react is `onClick={onCloseBook}`
> 
> for reference: https://upmostly.com/tutorials/react-onclick-event-handling-with-examples

It's actually very good practice to do this. You're avoiding additional javascript event firing and calls/parsing of the dom. As the game gets more complex script executions like that can snowball and hit performance.

Reply:
@ahh1539 
sure I can change this according to this but I just reused the previous code that was already there.
so if we are fixing this we need to fix the code previously as well, but not sure this needs to be in this PR